conans/test/functional/editable/forbidden_commands_test.py

```diff
@@ -24,6 +24,7 @@ def setUp(self):
         self.t = TestClient(servers=self.servers, users={"default": [("lasote", "mypass")]})
         self.t.save(files={'conanfile.py': self.conanfile,
                            CONAN_PACKAGE_LAYOUT_FILE: "", })
+        self.t.run('export . {}'.format(self.ref))
```

Comment:
why?

Reply:
Otherwise, there is nothing in the local cache to be removed, and the test doesn't raise, because it doesn't try to removing anything.

Reply:
Fair, I will decouple the offending test